src/markdown/careers/emea/senior-product-designer-es-2022-08-04.md

```diff
@@ -0,0 +1,77 @@
+---
+jobId: 'oMYwkfwk'
```

Comment:
can you double-check this job ID? the link it opens up is the BR position

Reply:
@plhnk thanks for finding that — I updated and checked for the correct link